vpc-sample/tests.cloudbuild.yaml

```diff
@@ -0,0 +1,108 @@
+steps:
+- id: Deploy Restricted Function
```

Comment:
Would it make sense to parallelize the steps for deploying the Cloud Function and Cloud Run service?

Reply:
I've updated it to run the Function Deploy and Build Container image steps in parallel